test/lib/StringUtilities.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { trainCase } from '../../src/lib/StringUtilities';
+
+import * as chai from 'chai';
+
+const should = chai.should();
+
+describe('StringUtilities', () => {
+    describe('#trainCase', () => {
+        it('should return undefined for an undefined value', () => {
```

Comment:
probably should add some cases for strings with non-alpha chars.  like `'1', '1-1'` and just `'-'`.